<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataPointViewer</class>
 <widget class="QWidget" name="dataPointViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Point Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkShowRaw">
       <property name="text">
        <string>Show Raw Data</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sensor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editSensor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Last Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="editDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butCleanup">
       <property name="text">
        <string>Cleanup...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butCompute">
       <property name="text">
        <string>Compute...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabDataPoint">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <widget class="QWidget" name="tabInfos">
      <attribute name="title">
       <string>Infos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="boxNotice">
           <property name="title">
            <string>Notice:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPlainTextEdit" name="textNotice">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="boxGeom">
           <property name="title">
            <string>Geometry:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPlainTextEdit" name="textGeometry">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="boxInstall">
         <property name="title">
          <string>Installation:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QScrollArea" name="scrollAreaInstall">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="widgetInstall">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>861</width>
               <height>526</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="labelInstall">
                <property name="text">
                 <string>Installation PNG Image</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTableRawTemp">
      <attribute name="title">
       <string>Raw Temp.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableViewRawTemp">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTableRawPress">
      <attribute name="title">
       <string>Raw Volt.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="tableViewRawPress">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlotRaw">
      <attribute name="title">
       <string>Raw Plot</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QScrollArea" name="scrollAreaPlotRaw">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="widgetPlotRaw">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>887</width>
            <height>756</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="layoutPlotRaw"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTemp">
      <attribute name="title">
       <string>Temperature</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="tableViewTemp">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPress">
      <attribute name="title">
       <string>Pressure</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableView" name="tableViewPress">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlot">
      <attribute name="title">
       <string>Plot</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QScrollArea" name="scrollAreaPlot">
         <widget class="QWidget" name="widgetPlot">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>861</width>
            <height>600</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetPlotTemp" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>300</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="layoutPlotTemp">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widgetPlotPress" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>300</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="layoutPlotPress">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabResults">
      <attribute name="title">
       <string>Results</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>887</width>
            <height>756</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetResTemp" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>300</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="layoutResTemp"/>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widgetResProf" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>300</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="layoutResProf"/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInversion">
      <attribute name="title">
       <string>Inversion</string>
      </attribute>
      <layout class="QVBoxLayout" name="layoutInversion"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
